cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -80,8 +86,15 @@ type renewConfig struct {
 	csrPath        string
 }
 
-func getRenewSubCommands() []*cobra.Command {
-	cfg := &renewConfig{}
+func getRenewSubCommands(kdir string) []*cobra.Command {
+	cfg := &renewConfig{
+		cfg: kubeadmapiv1beta2.InitConfiguration{
+			ClusterConfiguration: kubeadmapiv1beta2.ClusterConfiguration{
+				// Setting kubernetes version to a default value in order to allow a not necessary internet lookup
+				KubernetesVersion: fmt.Sprintf("v%s", kubeadmconstants.CurrentKubernetesVersion),
```

Comment:
It's a little weird that this is here.

Reply:
No need for `Sprintf` here, just `constants.CurrentKubernetesVersion.String()` should do the trick.

Reply:
yep, it is a bit weird, but ATM it is the workaround used to get commands which don't use the KubernetesVersion to work in air-gapped environments